Gardner.css

```diff
@@ -0,0 +1,386 @@
+*
+{
+    margin: 0;
+    padding: 0;
+    /* box-sizing allows us to include padding and border in element's total width and height  */
+    box-sizing: border-box;
+    text-decoration: none;
+}
+body
+{
+    background-color: #F7F6F4;
+    font-family: 'Manrope', sans-serif;
+}
+.navbar
+{
+    position: fixed;
+    width: 100%;
+    padding: 30px;
+    z-index: 999;
+    transition: all 0.3s ease;
+}
+.box
+{
+    width:60px;
+    background-color: #709D61;
+    float: left;
```

Comment:
Avoid using float. Bad for responsive design

Reply:
Okay